src/olympia/bandwagon/models.py

```diff
@@ -93,10 +93,6 @@ class Collection(ModelBase):
     addon_count = models.PositiveIntegerField(default=0,
                                               db_column='addonCount')
 
-    all_personas = models.BooleanField(
-        default=False,
-        help_text='Does this collection only contain Themes?')
```

Comment:
This needs a database migration, mind filing an issue for that?

Reply:
#11718